ember-phone-input/src/instance-initializers/phone-input.ts

```diff
@@ -0,0 +1,17 @@
+import type ApplicationInstance from '@ember/application/instance';
+import type PhoneInputService from '../services/phone-input';
+
+export function initialize(appInstance: ApplicationInstance): void {
+  const config = appInstance.resolveRegistration('config:environment') as
+    | { phoneInput: { lazyLoad: boolean } | undefined }
+    | undefined;
```

Comment:
Do you think is it worth to have the type specified in an `interface` to have this more readable?

Reply:
We have a lint rule that prevents us from using aliases which means we actually shouldn't create custom types just to make code cleaner